modules/shipit_scriptworker/manifests/settings.pp

```diff
@@ -55,32 +59,56 @@
             },
         },
       },
+      'tb-dev'  => {
+        worker_group             => 'shipit-v1',
+        worker_type              => 'tb-shipit-dev',
+        verbose_logging          => true,
```

Comment:
I think this is unused since in modules/shipit_scriptworker/manifests/init.pp you're using the one from line 111 - https://github.com/mozilla/build-puppet/pull/49/files#diff-16fc35f7191a6528caf61089d8af4f3eR111